src/WebJobs.Script.WebHost/Management/LinuxSpecialization/PackageCopyHandler.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.Azure.WebJobs.Script.Diagnostics;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Management.LinuxSpecialization
+{
+    public class PackageCopyHandler : IPackageCopyHandler
+    {
+        private readonly ILogger<PackageCopyHandler> _logger;
+
+        public PackageCopyHandler(ILogger<PackageCopyHandler> logger)
+        {
+            _logger = logger ?? throw new ArgumentNullException(nameof(logger));
+        }
+
+        public string CopyPackageFile()
+        {
+            var home = Environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteHomePath);
+            var packageFolderPath = Path.Combine(home, "data", "SitePackages");
```

Comment:
>"data" [](http://example.com/codeflow?start=55&length=6)

if we dont have constants defined already, can move to some constants in the right csproj

Reply:
I have introduced the constants.